configure.ac

```diff
@@ -1478,6 +1478,17 @@ then
 fi
 
 
+AC_ARG_ENABLE([armasm-sha512],
```

Comment:
Perhaps thinking long term for possible additional ARMv8.2 SHA instruction support (some of which are optional), would it make sense to support something like `--enable-armasm=sha512,sha3`?

* ARMv8.2-SHA SHA-3 instructions (EOR3, RAX1, XAR, BCAX) -> `WOLFSSL_ARMASM_CRYPTO_SHA3`
* ARMv8.2-SHA SHA512 instructions (SHA512) -> `WOLFSSL_ARMASM_CRYPTO_SHA512`